dts/arm/nxp/nxp_rt1060.dtsi

```diff
@@ -16,3 +16,29 @@
 	compatible = "mmio-sram";
 	reg = <0x20200000 0xC0000>;
 };
+
+/* i.MX rt1060 has a second Ethernet controller.
+ */
+/ {
+	aliases {
+		eth1 = &eth1;
+	};
+
+	soc {
```

Comment:
please `/delete-node/ &eth1;` in _boards/arm/mimxrt1060_evk/mimxrt1060_evk.dts_ and _mimxrt1064_evk_ 
SoC has 2 interfaces, but these boards don't have the second inteface wired.

Reply:
Done in 46e5bdddb4ecf414eb8c3c3d734a623741a11483